tests/strictequal.test.js

```diff
@@ -3,16 +3,29 @@ const it = require('mocha').it
 const expect = require('chai').expect
 const strictEqual = require("../strictequal")
 
-describe('Strict Equal Function', () => {
-    it('returns true if the values provided are strictly equal to each other', () => {
-        // set up test data
+describe('strictEqual', () => {
+    it('should return true if the numbers are the same type and value', () => {
         let a = 10
         let b = 10
-
-        //call the function from our outside file
-        strictEqual(a, b)
-
-        // assert what should be true
-        expect(a).to.equal(b)
+        const testData = strictEqual(a, b)
+        expect(testData).to.be.true
+    })
+    it('should return false if the numbers are not the same type', () => {
+        let a = '10'
+        let b = 10
```

Comment:
It is great learning deconstructed here is another shorter way to write it when you get more comfortable

it('returns true for 5 and 5', () => {
    expect(strictEqual(5, 5)).to.equal(true)
  })